### PR TITLE
Update fuel-asm to 0.6 and fuel-tx to 0.14

### PR DESCRIPTION
**Pull request by vlopes11**

### CODE REVIEW

**Comment by vlopes11** [comment]
Blocked by https://github.com/FuelLabs/fuel-tx/pull/151

**Comment by Dentosal** [comment]
@vlopes11 this should be unblocked now

**Review by Dentosal — APPROVED** [review]

**Review by Voxelot — APPROVED** [review]

**Comment by Voxelot** [comment]
seems like this should just get merged to unblock @Dentosal. Already has 3 approvals so I'll push the merge button.

**Pull request merged**
